### ISSUE TITLE
Error building on Mac

### ISSUE THREAD

**Issue by thomasahle** [google-research/google-research#507]
After running `configure.py` I try to build with Bazel, but I'm getting the following error:

    $ CC=clang bazel build -c opt --copt=-mavx2 --copt=-mfma --cxxopt="-D_GLIBCXX_USE_CXX11_ABI=0" --cxxopt="-std=c++17" --copt=-fsized-deallocation --copt=-w :build_pip_pkg
    INFO: Build option --action_env has changed, discarding analysis cache.
    WARNING: /private/var/tmp/_bazel_thdy/2bcb61de71a5e80e9c149676ae8deec4/external/local_config_tf/BUILD:5430:8: target 'ensorflow_framework.2' is both a rule and a file; please choose another name for the rule
    ERROR: /private/var/tmp/_bazel_thdy/2bcb61de71a5e80e9c149676ae8deec4/external/local_config_tf/BUILD:11:11: in srcs attribute of cc_library rule @local_config_tf//:libtensorflow_framework: '@local_config_tf//:ensorflow_framework.2' does not produce any cc_library srcs files (expected .cc, .cpp, .cxx, .c++, .C, .cu, .cl, .c, .h, .hh, .hpp, .ipp, .hxx, .h++, .inc, .inl, .tlh, .tli, .H, .tcc, .S, .s, .asm, .a, .lib, .pic.a, .lo, .lo.lib, .pic.lo, .so, .dylib, .dll, .o, .obj or .pic.o)
    ERROR: Analysis of target '//:build_pip_pkg' failed; build aborted: Analysis of target '@local_config_tf//:libtensorflow_framework' failed
    INFO: Elapsed time: 0,692s
    INFO: 0 processes.
    FAILED: Build did NOT complete successfully (5 packages loaded, 319 targets configured)
        Fetching ...ogle_protobuf; Cloning tags/v3.9.2 of https://github.com/protocolbuffers/protobuf.git

This is on

    $ uname -a
    Darwin mac611211 19.6.0 Darwin Kernel Version 19.6.0: Mon Aug 31 22:12:52 PDT 2020; root:xnu-6153.141.2~1/RELEASE_X86_64 x86_64

**Comment by phrasenmaeher** [comment]
It has been some time since you've opened this issue. I am now also having it, did you manage  to find any solution?

**Comment by thomasahle** [comment]
I don't think so, I'm afraid, but I've actually forgotten what I was trying to do...
